pixeltable/dataframe.py

```diff
@@ -195,10 +236,10 @@ def collect(self) -> DataFrameResultSet:
         except sql.exc.DBAPIError as e:
             raise exc.Error(f'Error during SQL execution:\n{e}')
 
-        col_names = [expr.display_name() for expr in self.select_list]
-        # replace ''
-        col_names = [n if n != '' else f'col_{i}' for i, n in enumerate(col_names)]
-        return DataFrameResultSet(data_rows, col_names, [expr.col_type for expr in self.select_list])
+        expanded_select_list = _expand_select_list(self.tbl, self.select_list)
```

Comment:
Given that the caller always separates the exprs from the aliases, might be good to return them separately.